### PR TITLE
Bug/set real data reduce post

### PR DESCRIPTION
**Pull request by deonjo**
Reduce a call to post when setting data, should only post when saving.

Fixed all calls. Should also help a little with concurrent loads of the page, so it won't overwrite.

### CODE REVIEW

File: src/reducers/real.jsx [hunk 1]
```diff
@@ -152,7 +152,6 @@ const realReducer = (state = initialState, action) => {
         }
 
         case SET_REAL_DATA: {
-            exportState(action.data)
             return cloneDeep(action.data)
```
Comment (deonjo) [inline src/reducers/real.jsx]: There's no longer an export at this step.

File: src/reducers/real.jsx [hunk 2]
```diff
@@ -152,7 +152,6 @@ const realReducer = (state = initialState, action) => {
         }
 
         case SET_REAL_DATA: {
```
Comment (crystaldou) [inline src/reducers/real.jsx]: can we rename this reducer too? since u r renaming everything else?

Comment (crystaldou) [reply src/reducers/real.jsx]: umm i'm so confused by all these names.. @_@;;

Comment (deonjo) [reply src/reducers/real.jsx]: I'll do a large refactor after the retreat.

**Pull request merged**